src/Router.js

```diff
@@ -13,8 +14,9 @@ const Router = () => {
         <Route path="/" element={<Main />} />
         <Route path="/login" element={<Login />} />
         <Route path="/signup" element={<Signup />} />
-        <Route path="/bestProductList" element={<BestProductList />} />
-        <Route path="/productDetail" element={<ProductDetail />} />
+        <Route path="/bestproductList" element={<BestProductList />} />
+        <Route path="/productdetail/:id" element={<ProductDetail />} />
+        <Route path="/productlist" element={<ProductList />} />
```

Comment:
path url을 나타낼 때는 kebab-case를 활용합니다
```suggestion
        <Route path="/best-product-list" element={<BestProductList />} />
        <Route path="/product-detail/:id" element={<ProductDetail />} />
        <Route path="/product-list" element={<ProductList />} />
```
등등으로 수정해주세요!